src/packet/one_pass_signature.js

```diff
@@ -40,8 +38,8 @@ class OnePassSignaturePacket {
   }
 
   constructor() {
-    /** A one-octet version number.  The current version is 3. */
-    this.version = null;
+    /** A one-octet version number.  The current versions are 3 and 6. */
+    this.version = 3;
```

Comment:
We're moving away from initialising values in the constructor that are later overwritten (either by `read` or when signing with a v6 key in this case), so I'd leave this null (same as with signature packets)

Reply:
Makes sense. I copied it from the PKESK packet implementation. I will change it there as well.

Reply:
Ok, it seems that this change breaks some tests. So, I will leave it a 3.

Reply:
The fact that it breaks some tests is a bad sign, since it was `null` to begin with 😓  there might be a missing overwrite based on key version

Reply:
The issue was that objects were initialised with the constructor without setting the version field. Fixed now.